dist/types/detailedIdentity.js

```diff
@@ -0,0 +1,79 @@
+"use strict";
+const Group = require('./group');
+const RoleMapping = require('./roleMapping');
+/**
+ * Detailed information about a registered Identity for a Tozny realm.
+ */
+class DetailedIdentity {
+    constructor(id, username, email, firstName, lastName, active, federated, roles, groups, attributes) {
+        this.id = id;
+        this.username = username;
+        this.email = email;
+        this.firstName = firstName;
+        this.lastName = lastName;
+        this.active = active;
+        this.federated = federated;
+        this.roles = roles;
```

Comment:
is this missing a client section?

Reply:
everything in the `dist` directory can be ignored from a code review perspective